pages/common/tgpt.md

```diff
@@ -0,0 +1,37 @@
+# tgpt
+
+> Talk to an AI chatbot without the need for API keys.
+> Available providers: `openai`, `opengpts`, `koboldai`, `phind`, `llama2`, `blackboxai`.
+> More information: <https://github.com/aandrew-me/tgpt>.
+
+- Chat with the default provider (GPT-3.5-turbo):
+
+`tgpt "{{prompt}}"`
+
+- Start [m]ulti-line interactive mode:
+
+`tgpt --multiline`
+
+- Generate [i]mages and save them to the current directory:
+
+`tgpt --image "{{prompt}}"`
+
+- Generate [c]ode with the default provider (GPT-3.5-turbo):
+
+`tgpt --code "{{prompt}}"`
+
+- Chat with a specific provider without animations [q]uietly:
```

Comment:
This sounds as if "quietly" meant something more than "without animations".
```suggestion
- Chat with a specific provider [q]uietly (without animations):
```

Reply:
oh right, will update